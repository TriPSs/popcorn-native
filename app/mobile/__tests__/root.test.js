import * as React from 'react'
import { shallow } from 'enzyme'

import Root from '../root'

describe('(Mobile) Root', () => {

  it('should match generated snapshot', () => {
    const wrapper = shallow(<Root />)
    expect(wrapper).toMatchSnapshot()
  })

})
