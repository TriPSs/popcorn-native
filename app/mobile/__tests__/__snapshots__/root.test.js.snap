// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Mobile) Root should match generated snapshot 1`] = `
<ForwardRef(NavigationContainer)>
  <Disclaimer>
    <IpFinder>
      <Component />
    </IpFinder>
  </Disclaimer>
</ForwardRef(NavigationContainer)>
`;
